hack/lib/util/golang.sh

```diff
@@ -11,8 +11,9 @@ function os::golang::verify_go_version() {
 	if [[ "${go_version[2]}" != go1.8* ]]; then
```

Comment:
@stevekuznetsov Should this be using `OS_REQUIRED_GO_VERSION` instead of hard-coding it to 1.8?

I'm guessing changes to this file will be overwritten when we sync the tooling again.

Reply:
Yes. A refactor went in to do that, but maybe it was in Origin and not in `release`?

Reply:
Not that I see: https://github.com/openshift/release/blob/master/tools/hack/golang/hack/lib/util/golang.sh#L11